tests/NuGet.IndexingTests/TestSupport/Constants.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Lucene.Net.Search;
+using NuGet.Indexing;
+
+namespace NuGet.IndexingTests.TestSupport
```

Comment:
I don't mind moving these into the NuGet.IndexingTests namespace, but the other files in the TestSupport folder are already in the TestSupport namespace. Do we just want to get rid of that entirely and put everything in the INdexingTests namespace?